pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -1087,9 +1086,6 @@ public void addTable(@Nonnull TableConfig tableConfig)
         // lets add table configs
         ZKMetadataProvider.setOfflineTableConfig(_propertyStore, tableNameWithType, tableConfig.toZNRecord());
 
-        _propertyStore.create(ZKMetadataProvider.constructPropertyStorePathForResource(tableNameWithType),
```

Comment:
Why don't we need to write to property store when adding a table?

Reply:
yeah, and all the tests passed too!

Reply:
We don't need to initialize this directory because it will be automatically created when the first segment gets pushed. Real-time table does not initialize this directory, so keep them consistent